{{cookiecutter.app_name}}/.env.test

```diff
@@ -0,0 +1 @@
+DATABASE_URL=postgres://postgres:postgres@0.0.0.0:5433/{{cookiecutter.app_name}}_test?sslmode=disable
```

Comment:
`DATABASE_URL` does not seem to be the variable that must be stored. First, it's super long and there can be typo/errors. The only differences between these two variables are the app name, the DB name and the port 🤔  I understand the issue you raised as this needs to be read by the TOML file but it looks like the wrong directiomn.

Reply:
Sorry, I'm lost. Could you please explain more about this?
What is your suggestion?

Reply:
I don't have a suggestion. It's just that having such a long string as an environment variable is not great. It can work though. So it can stay like this of there are no better options.